de.metas.swat/de.metas.swat.base/src/test/java/de/metas/invoicecandidate/api/impl/InvoiceCandDAOTest.java

```diff
@@ -127,17 +127,25 @@ public void updateMissingPaymentTermIds()
 	@Test
 	public void testInvoiceCandidates()
 	{
+		createInvoiceCandidate(EXTERNAL_LINE_ID1, EXTERNAL_HEADER_ID1);
+		createInvoiceCandidate(EXTERNAL_LINE_ID2, EXTERNAL_HEADER_ID2);
+		createInvoiceCandidate(EXTERNAL_LINE_ID3, EXTERNAL_HEADER_ID3);
 		ExternalId externalId1 = ExternalId.of(EXTERNAL_LINE_ID1);
 		ExternalId externalId2 = ExternalId.of(EXTERNAL_LINE_ID2);
-		ExternalId externalId3 = ExternalId.of(EXTERNAL_LINE_ID3);
-		List<ExternalId> externalLineIds = new ArrayList<ExternalId>();
-		externalLineIds.add(externalId1);
-		externalLineIds.add(externalId2);
-		externalLineIds.add(externalId3);
-		List<ExternalHeaderAndLineId> headerAndLineIds = new ArrayList<ExternalHeaderAndLineId>();
-		headerAndLineIds.add(ExternalHeaderAndLineId.builder().externalHeaderId(EXTERNAL_HEADER_ID1).externalLineIds(externalLineIds).build());
-		IQuery<I_C_Invoice_Candidate> createQueryByHeaderAndLineId = new InvoiceCandDAO().createQueryByHeaderAndLineId(headerAndLineIds);
+
+		List<ExternalId> externalLineIds1 = new ArrayList<ExternalId>();
+		externalLineIds1.add(externalId1);
+		List<ExternalId> externalLineIds2 = new ArrayList<ExternalId>();
+		externalLineIds2.add(externalId2);
+
+		List<ExternalHeaderAndLineId> headerAndLineIds1 = new ArrayList<ExternalHeaderAndLineId>();
+		headerAndLineIds1.add(ExternalHeaderAndLineId.builder().externalHeaderId(EXTERNAL_HEADER_ID1).externalLineIds(externalLineIds1).build());
+		
+		headerAndLineIds1.add(ExternalHeaderAndLineId.builder().externalHeaderId(EXTERNAL_HEADER_ID2).externalLineIds(externalLineIds2).build());
+		
+		IQuery<I_C_Invoice_Candidate> createQueryByHeaderAndLineId = new InvoiceCandDAO().createQueryByHeaderAndLineId(headerAndLineIds1);
 		int size = createQueryByHeaderAndLineId.list().size();
+		assert(size==3);
```

Comment:
i think this is the java-1.4 assert keyword. not a junit assert.
please use assertJ instead (like in the `updateMissingPaymentTermIds()` test  in this class)